plan.sh

```diff
@@ -35,6 +35,11 @@ do_prepare() {
   build_line "Setting CC=$CC"
 }
 
+do_build() {
+  ./configure --host=arm
```

Comment:
I'm not super familiar but this looks like you are telling it to cross compile for arm?

Reply:
yes matt